actions/sequester/Quest2GitHub/QuestGitHubService.cs

```diff
@@ -237,26 +237,11 @@ await foreach (var label in labelQuery.AllLabels())
                 Value = ghIssue.IsOpen ? "Active" : "Closed",
             });
         }
-        while (patchDocument.Any())
+        if (patchDocument.Any())
         {
             var jsonDocument = await _azdoClient.PatchWorkItem(questItem.Id, patchDocument);
-            try
-            {
-                var newItem = QuestWorkItem.WorkItemFromJson(jsonDocument);
-                return newItem;
-            }
-            catch (KeyNotFoundException)
-            {
-                // This will happen when the assignee IS a Microsoft FTE,
-                // but that person isn't onboarded in Quest. (Likely a product team member, or CSE)
-
-                // So, remove the node with the Assigned to and try again:
-                if (!patchDocument.Remove(assignPatch!))
-                {
-                    // If there's no assign node, clear the doc and punt
-                    patchDocument.Clear();
-                }
-            }
+            var newItem = QuestWorkItem.WorkItemFromJson(jsonDocument);
+            return newItem;
```

Comment:
This set of changes is a simplification now that we're first querying for an FTE and matching them in Quest. We shouldn't be trying to assign an issue to a person who isn't a user in our AzDo instance. So, that part of the update shouldn't fail. 

If it does, we did something wrong.